tensorflow.md

```diff
@@ -1,32 +1,77 @@
-# Building TensorFlow on Graviton2
+# ML inference on Graviton CPUs with TensorFlow
 
-Install bazel build system:
+There are two ways to use tensforflow based ml inferencing on Graviton CPUs.
+- Tensorflow DLAMI
+- Tensorflow Docker container
+
+**Using AWS DLAMI:**
+To launch a Graviton instance with a DLAMI via EC2 console:
+
+1. Click on "Community AMIs"
+2. Filter AMIs by clicking Ubuntu and Arm (64-bit) check boxes
+3. Search for "Deep Learning" in search field to list the current Ubuntu based DLAMIs"
+4. Select the AMI: "Deep Learning AMI Graviton Tensorflow <version>(Ubuntu 20.04) <yyyy/mm/dd>"
+
+Here is the awscli snippet to launch the same:
+
+aws ec2 run-instances --region us-west-2 --image-id ami-0e273fef68b695319  --instance-type c6g.4xlarge --count 1  --key-name <key_name_to_added> --subnet-id subnet-1bafd750 --block-device-mappings 'DeviceName=/dev/sda1,Ebs={VolumeSize=256}' --tag-specifications 'ResourceType=volume,Tags=[{Key=Name,Value="Graviton TF DLAMI"},{Key=< >,Value=< >}]' 'ResourceType=instance,Tags=[{Key=Name,Value="Graviton TF DLAMI"},{Key=< >,Value=< >}]'
```

Comment:
this is doing a lo more. You don't need to generally specify a subnet, and their subnet certainly isn't this one. Same with tags specified, that is really up to the customer.